src/lib/FullCalendar.svelte

```diff
@@ -124,11 +124,10 @@
 	<article>
 		<h2>Add to Your Calendar</h2>
 		<p>
-			Copy the iCal url into your calendar app to automatically add Kendo Club events to your
-			calendar.
+			Copy this iCal url into your calendar app to subscribe to the Kendo Club at Umich calendar.
 		</p>
 		<div role="group">
-			<input value={icalUrl} readonly />
+			<input value={'https://' + icalUrl} readonly />
```

Comment:
I usually prefer to do `https://{icalUrl}` but I guess I this is fine.

Reply:
I'm not completely sure how to do that since this is inside of a script tag

Reply:
It's not inside a script tag

Reply:
Oh you're right, oops